src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -3064,14 +3060,16 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_common_scenarios_workflow_dashboard::misc_test_0002);
     REGISTER_TEST(GUITest_common_scenarios_workflow_dashboard::misc_test_0003);
     REGISTER_TEST(GUITest_common_scenarios_workflow_dashboard::misc_test_0004);
+
+    REGISTER_TEST_L(GUITest_common_scenarios_workflow_dashboard::tree_nodes_creation_test_0001, labels({Metagenomics, Linux})); // Needs full taxonomy data (Not available in Nightly).
```

Comment:
As far as I know, the "Metagenomics" suite doesn't use this labels right now, test should be added to "test/general/metagenomic_gui_tests_list.txt"

Reply:
Ok, I will add tests to that list